src/styles/product-card.scss

```diff
@@ -0,0 +1,170 @@
+// #region variables //
+// Variables for colors and spacing
+$main-accent: #001f3f;
+$secondary: #999;
+$white: #fff;
+$blue-accent: #00acdc;
+$gray-elements: #ccc;
+$yellow-stars: #fc0;
+$font-family: 'Roboto', sans-serif;
+$white-accent: #f3f3f3;
+
+// #endregion //
+
+// #region catalog //
+@media (min-width: 1200px) {
+  .catalog-container {
+    margin: auto;
+  }
+}
+
+.catalog {
+  justify-content: center;
+  padding: 50px 0;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+}
+
+// #endregion //
+
+// #region card //
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-family;
+  background-color: $white;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: #060b35;
```

Comment:
Consider using a variable for the color `#060b35` to maintain consistency and make future changes easier.